keyboards/checkerboards/quark_lp/keymaps/via/rules.mk

```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes

```

Comment:
As above.

Reply:
excuse the ignorance - what needs to be changed here?

Reply:
Your info.json and this file are using Windows line endings - I can tell because when I make a suggestion on a single line, GitHub shows this:
![image](https://user-images.githubusercontent.com/4781841/134288421-9d2ff6e0-197d-4231-804b-3228b4fe15d4.png)

Reply:
![github-quark-lp](https://user-images.githubusercontent.com/40127181/134289474-13d9f237-b817-49f6-bcdb-7213e9fb63a1.jpg)

Sorry again, I don't know what I'm looking for here. How do I change to Unicode?

Reply:
You need to use a proper text editor, like Sublime or VS Code. It will allow you to change the line endings from CRLF to LF.

Reply:
I used Atom to create the info.json and VIA rule files. They appear to be working on my end. I checked the json in qmk.config.fm and the VIA through the design tab in the app. 

But I'm happy to change the files if this is not the case.

Reply:
Yes, the files themselves work fine, the issue is that the line endings are not correct. QMK uses Unix line endings, not Windows, and if this PR were to be merged as is, it would cause these files to appear to users as modified, even though they aren't really, because Git is expecting them to use Unix line endings.

Reply:
files deleted and reuploaded with LF endings